src/Bridge/Doctrine/Common/Util/IdentifierManagerTrait.php

```diff
@@ -74,7 +74,7 @@ private function normalizeIdentifiers($id, ObjectManager $manager, string $resou
 
             $identifier = null === $identifiersMap ? $identifierValues[$i] ?? null : $identifiersMap[$propertyName] ?? null;
             if (null === $identifier) {
-                throw new PropertyNotFoundException(sprintf('Invalid identifier "%s", "%s" was not found.', $id, $propertyName));
+                throw new PropertyNotFoundException(sprintf('Invalid identifier "%s", "%s" was not found for resource: "%s".', $id, $propertyName, $resourceClass));
```

Comment:
We tend to use the short name instead of the whole class (it gives less information to an attacker too).